src/app/community-view/community-view.component.ts

```diff
@@ -104,6 +114,15 @@ export class CommunityViewComponent implements OnInit {
     });
   }
 
+  openInvitationDialog() {
+    this.dialog.open(CommunityInvitationDialogComponent, {
```

Comment:
We could subscribe here to dialog closed event with `dialogRef.afterClosed()`, receive `CommunityUserRegistrationResponse[]` and display a notification telling the authenticated user something like **Users John, Tom, Dan were successfully invited** to join the community.

Reply:
thank you. done